MOONCLOCK/apps.py

```diff
@@ -2,14 +2,15 @@
 import time
 
 from adafruit_datetime import datetime
-from utils import get_current_datetime, timestamp_to_time, str_rjust, str_cjust
+from utils import get_current_datetime, timestamp_to_time, str_align
 
 
 class App:
-    def __init__(self, display_group, requests, duration=0, update_frequency=None):
+    def __init__(self, display_group, requests, duration=1, align='right', update_frequency=None):
```

Comment:
Do we want some default duration?

Reply:
We had a default value of `duration`, it was `0`. This is because of "one hit" applications that are supposed to end asap.